components/spotlight/SpotlightTable.vue

```diff
@@ -59,8 +59,9 @@
       </b-table-column>
 
       <b-table-column
-        field="uniqueCollectors"
-        :label="$t('spotlight.uniqueCollectors')">
+        field="unique_collectors"
```

Comment:
`unique_collectors` sounds like something we should hack in [kodadot/rubick](https://github.com/kodadot/rubick)